practice/react-practice-one/src/components/Toasts/Toast.jsx

```diff
@@ -0,0 +1,26 @@
+import { useEffect } from 'react';
+
+/* Import dependencies */
+import PropTypes from 'prop-types';
+
+/* Import CSS */
+import './toast.css';
+
+const Toast = ({ message, type, duration = 3000, onClose }) => {
+  useEffect(() => {
+    const timer = setTimeout(onClose, duration);
+
+    return () => clearTimeout(timer);
+  }, [duration, onClose]);
+
+  return <div className={`toast ${type}`}>{message}</div>;
+};
+
+Toast.propTypes = {
+  message: PropTypes.string.isRequired,
+  type: PropTypes.string.isRequired,
```

Comment:
@JennyDoJD can we set a free text for `type`?

Reply:
yes

Reply:
did you try it? How does the UI look?

Reply:
![image](https://github.com/user-attachments/assets/6c00d90a-b89c-492d-b7af-15efbca12e71)

![image](https://github.com/user-attachments/assets/a07a44a3-562f-4d30-b5f2-3ba84155f484)